andela/hc-kalii
# #153727982 Add  better description for the nag feature on the frontend

#### What does this PR do?
* Adds better description for what the nag feature does on the frontend
#### How should this be manually tested?
* Execute `./manage.py runserver`
* Login with a valid email address and confirm login from your Inbox.
#### What are the relevant pivotal tracker stories?
* [153727982](https://www.pivotaltracker.com/story/show/153727982)
#### Questions
